src/NzbDrone.Core/Parser/Parser.cs

```diff
@@ -940,15 +940,14 @@ public static string ParseReleaseGroup(string title)
         public static string RemoveFileExtension(string title)
         {
             title = FileExtensionRegex.Replace(title, m =>
+            {
+                var extension = m.Value.ToLower();
+                if (MediaFiles.MediaFileExtensions.Extensions.Contains(extension) || new[] { ".par2", ".nzb" }.Contains(extension))
                 {
-                    var extension = m.Value.ToLower();
-                    if (MediaFiles.MediaFileExtensions.Extensions.Contains(extension) || new[] { ".par2", ".nzb" }.Contains(extension))
-                    {
-                        return string.Empty;
-                    }
-
-                    return m.Value;
-                });
+                    return string.Empty;
+                }
+                return m.Value;
```

Comment:
Keep the new line above the return to separate them visually.

```suggestion

                return m.Value;
```